src/components/containers/TraceAccordion.js

```diff
@@ -91,19 +168,34 @@ class TraceAccordion extends Component {
             </TabList>
             <TabPanel>
               <PlotlyPanel>
-                {individualTraces ? individualTraces : null}
+                {filteredTraces.length
```

Comment:
this ternary is reused in a few places. Let's push it into `renderIndividualTraceFolds` to make it a bit more robust, and then we can make this code more readable.